offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/overview/projectcreation/CreateProjectView.groovy

```diff
@@ -148,37 +148,37 @@ class CreateProjectView extends VerticalLayout{
 
     private void createProjectSpaceElements() {
         // Set a nice header
-        Label label = new Label("1. Please select/create a project space first")
+        Label label = new Label("1. Select/Create Project Space")
         label.setStyleName(ValoTheme.LABEL_H3)
         this.inputFields.addComponent(label)
 
         /* The user needs to choose between creating a new project space
          or select an existing one */
         // First we create a ratio group with the choices available
-        projectSpaceSelection = new RadioButtonGroup<>("Create project in",
+        projectSpaceSelection = new RadioButtonGroup<>("Create Project In",
```

Comment:
Since this sentence will be completed I would ignore the cases here (see my comment and screenshot above)

```suggestion
        projectSpaceSelection = new RadioButtonGroup<>("Create project in",
```

Reply:
makes sense